nirc_ehr/src/org/labkey/nirc_ehr/dataentry/DeathFormType.java

```diff
@@ -0,0 +1,43 @@
+package org.labkey.nirc_ehr.dataentry;
+
+import org.labkey.api.ehr.EHRService;
+import org.labkey.api.ehr.dataentry.AnimalDetailsFormSection;
+import org.labkey.api.ehr.dataentry.DataEntryFormContext;
+import org.labkey.api.ehr.dataentry.SimpleGridPanel;
+import org.labkey.api.ehr.dataentry.TaskFormSection;
+import org.labkey.api.ehr.dataentry.UnsaveableTask;
+import org.labkey.api.ehr.dataentry.forms.DeathInstructionsFormSection;
+import org.labkey.api.ehr.security.EHRCompletedInsertPermission;
+import org.labkey.api.ehr.security.EHRSurgeryEntryPermission;
+import org.labkey.api.module.Module;
+
+import java.util.Arrays;
+
+/**
+ * User: bimber
+ * Date: 7/29/13
+ * Time: 5:03 PM
+ */
+public class DeathFormType extends UnsaveableTask
+{
+    public static final String NAME = "death";
+
+    public DeathFormType(DataEntryFormContext ctx, Module owner)
+    {
+        super(ctx, owner, NAME, "Death", "Colony Management", Arrays.asList(
+                new DeathInstructionsFormSection(),
+                new TaskFormSection(),
+                new AnimalDetailsFormSection(),
+                new SimpleGridPanel("study", "deaths", "Deaths")
+        ));
+    }
+
+    @Override
+    protected boolean canInsert()
```

Comment:
I think you can delete this override. We can adjust as needed in the future, but no need to copy this specific rule from another center.